src/lib.rs

```diff
@@ -1371,15 +1366,6 @@ impl LUT {
         redundant_many_lwe
     }
 
```

Comment:
Why this function is removed ?

    fn add_redundancy(
        lwe: &LweCiphertext<Vec<u64>>,
        ctx: &Context,
    ) -> Vec<LweCiphertext<Vec<u64>>> {
        let box_size = ctx.box_size();
        let redundant_lwe: Vec<LweCiphertext<Vec<u64>>> = vec![(*lwe).clone(); box_size];
        redundant_lwe
    }

Reply:
because it's dead code